doc/source/development/contributing.rst

```diff
@@ -135,9 +135,67 @@ operations. To install pandas from source, you need to compile these C
 extensions, which means you need a C compiler. This process depends on which
 platform you're using.
 
-* Windows: https://devguide.python.org/setup/#windows-compiling
-* Mac: https://devguide.python.org/setup/#macos
-* Unix: https://devguide.python.org/setup/#unix-compiling
+**Windows**
+
+The building command depends on the architecture of the Python interpreter,
+32-bit or 64-bit. You can check the architecture by running the following in
+``cmd`` or ``powershell`` console::
+
+    python -c "import struct; print(struct.calcsize('P') * 8)"
+
+The above commands assume that you have the Python installation folder in your
+PATH environment variable.
+
+You will need `Build Tools for Visual Studio 2017
+<https://visualstudio.microsoft.com/downloads/>`_.
+
+.. warning::
+	You DO NOT need to install Visual Studio 2019.
+	You only need "Build Tools for Visual Studio 2019" found by
+	scrolling down to "All downloads" -> "Tools for Visual Studio 2019".
+
+For 64-bit Python, configure the build environment with::
+
+    SET DISTUTILS_USE_SDK=1
+    "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
```

Comment:
I *think* this step is not needed for pandas, as we use setuptools and not distutils as scikit-learn does (based on https://wiki.python.org/moin/WindowsCompilers#Distutils_notes, at least the env variables is not needed for setuptools)

Reply:
cc @chris-b1